### PR TITLE
Sales folder topic tune up  GH 725 - cart and checkout topics

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is one of a series to address #725  performs a "content tuneup" for the topics in the sales folder for the cart and checkout areas, with a few related items. There were a few topics that were consolidated and reorganized. These updates also addressed the following:

- Captions using sentence case
- Adding alt text for images
- Headings using sentence case and not infinitives
- Consistency in numbered step structure and style
- Edits for clarity
- Screen updates to eliminate the "cartoon balloons"

There are also a few places where links were updated for the consolidated/removed pages. I also found a missing option for the sales > sales config that had not been documented. 

Note: This will be applied first to the 2.3.x production docs and then forward ported.

Internal staging build at: 541/user-guide/v2.3/

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Affected documentation pages

- https://docs.magento.com/user-guide/v2.3/configuration/sales/sales.html
- https://docs.magento.com/user-guide/v2.3/sales/batches.html
- https://docs.magento.com/user-guide/v2.3/sales/billing-agreements.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-configuration.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-link.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-persistent-configuration.md
- https://docs.magento.com/user-guide/v2.3/sales/cart-persistent-workflow.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-persistent.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-quote-lifetime.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-redirect.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-sidebar.html
- https://docs.magento.com/user-guide/v2.3/sales/cart-thumbnails.html
- https://docs.magento.com/user-guide/v2.3/sales/cart.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-address-search.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-configuration.html (deleted, consolidated with checkout-options)
- https://docs.magento.com/user-guide/v2.3/sales/checkout-guest.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-instant-purchase.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-onepage.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-options.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-order-confirmation.html (deleted, consolidated with checkout-process)
- https://docs.magento.com/user-guide/v2.3/sales/checkout-payment-failed-emails.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-process.html
- https://docs.magento.com/user-guide/v2.3/sales/checkout-step1-shipping.html (deleted, consolidated with checkout-process)
- https://docs.magento.com/user-guide/v2.3/sales/checkout-step2-payment-review.html (deleted, consolidated with checkout-process)
- https://docs.magento.com/user-guide/v2.3/sales/checkout-totals-sort-order.html
- https://docs.magento.com/user-guide/v2.3/sales/minimum-order-amount.md (file renamed to cart-minimum-order-amount for consistency)
- https://docs.magento.com/user-guide/v2.3/sales/reorders-allow.html

### CODE REVIEW

File: src/sales/cart-persistent.md [hunk 1]
```diff
@@ -2,14 +2,17 @@
 title: Persistent Cart
 ---
 
-A persistent shopping cart keeps track of unpurchased items which are left in the cart, and saves the information for the customer’s next visit. Customers who are _remembered_ can have the contents of their shopping carts restored the next time they visit your store.
+A persistent shopping cart keeps track of unpurchased items that are left in the cart, and saves the information for the customer’s next visit. Customers who are _remembered_ can have the contents of their shopping carts restored the next time they visit your store.
```
Comment (hickskenh) [inline src/sales/cart-persistent.md:5]:
```suggestion
A persistent shopping cart keeps track of unpurchased items that are left in the cart and saves the information for the customer’s next visit. Customers who are _remembered_ can have the contents of their shopping carts restored the next time they visit your store.
```

File: src/sales/cart-persistent.md [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
+When using a persistent cart, we recommend that you set the lifetime of the server session and the session cookie to a long period of time. See [Session Lifetime]({% link customers/customer-online-options.md %}) for more information.
 
 To use the persistent shopping cart, the customer’s browser must be set to allow cookies. There are two types of cookies used for shopping cart operations:
 
 - **Session cookie** – A short-term session cookie exists for the duration of a single visit to your site, and expires when the customer leaves, or after a set period of time.
 
-- **Persistent Cookie** – A long-term persistent cookie continues in existence after the end of the session, and saves a record of the customers’ shopping contents for future reference.
-
-When using a persistent cart, we recommend that you set the lifetime of the server session and the session cookie to a long period of time. See [Session Lifetime]({% link customers/customer-online-options.md %}).
+- **Persistent Cookie** – A long-term persistent cookie continues in existence after the end of the session, and saves a record of the customer's shopping cart contents for future reference.
```
Comment (hickskenh) [inline src/sales/cart-persistent.md:18]:
```suggestion
- **Persistent Cookie** – A long-term, persistent cookie continues in existence after the end of the session and saves a record of the customer's shopping cart contents for future reference.
```

File: src/sales/checkout-address-search.md [hunk 3]
```diff
@@ -10,38 +10,39 @@ Address search is not enabled by default. You must configure this feature to inc
...
-_Select Shipping Address_
+![Luma storefront checkout with saved shipping addresses displayed]({% link images/images-ee/storefront-checkout-address-search.png %}){: .zoom}
+_Selecting a shipping address during checkout_
 
-If the customer does not have a default shipping address, the Shipping page displays "No address selected" and the customer must click **Change Address** to select a saved address or click <span class="btn">New Address</span> to add and select an address before proceeding with the checkout. If the customer does not have a default billing address, the Review & Payments page displays the address selected for shipping along with the Change Address option.
+If the customer does not have a default shipping address, the _Shipping_ page displays "No address selected" and the customer must click **Change Address** to select a saved address or click <span class="btn">New Address</span> to add and select an address before proceeding with the checkout. If the customer does not have a default billing address, the _Review & Payments_ page displays the address selected for shipping along with the _Change Address_ option.
```
Comment (hickskenh) [inline src/sales/checkout-address-search.md:16]:
```suggestion
If the customer does not have a default shipping address, the _Shipping_ page displays _No address selected_, and the customer must click **Change Address** to select a saved address or click <span class="btn">New Address</span> to add and select an address before proceeding with the checkout. If the customer does not have a default billing address, the _Review & Payments_ page displays the address selected for shipping along with the _Change Address_ option.
```

File: src/sales/checkout-address-search.md [hunk 4]
```diff
@@ -10,38 +10,39 @@ Address search is not enabled by default. You must configure this feature to inc
...
+   [_Sales configuration - checkout options_]({% link configuration/sales/checkout.md %})
 
 1. Set **Enable Address Search** to `Yes`.
 
 1. Set the **Number of Customer Addresses Limit** option to specify the threshold for including the address search feature.
 
    If necessary, clear the **Use system value** checkbox to make this change.
 
-   When the customer's number of saved addresses meets or exceeds this limit, the page displays either the default address (if the customer has one) or "No address selected" with the **Change Address** option. The default limit is 10.
+   When the customer's number of saved addresses meets or exceeds this limit, the page displays either the default address (if the customer has one) or "No address selected" with the _Change Address_ option. The default limit is `10`.
```
Comment (hickskenh) [inline src/sales/checkout-address-search.md:46]:
```suggestion
   When the customer's number of saved addresses meets or exceeds this limit, the page displays either the default address (if the customer has one) or _No address selected_ with the _Change Address_ option. The default limit is `10`.
```

File: src/sales/reorders-allow.md [hunk 5]
```diff
@@ -2,30 +2,33 @@
...
+## Criteria for reorder to be allowed for an order
 
-1. On the Admin sidebar, click **Stores**.
+- The _Allow Reorder_ configuration option must be enabled.
 
-1. In the _Settings_ section, choose **Configuration**.
+- If order is in `Hold` status, the reorder option is disabled.
 
-1. In the _Sales_ section in the left panel, choose **Sales**.
+- If one of the items in the order is unavailable, out of stock, or disabled, the reorder option is disabled.
```
Comment (hickskenh) [inline src/sales/reorders-allow.md:16]:
```suggestion
- If any of the items in the order is unavailable, out of stock, or disabled, the reorder option is disabled.
```

File: src/stores/attributes-customer-address.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ ee_only: true
 title: Customer Address Attributes
 ---
 
-The Customer Address Attribute determines the properties of street addresses that are entered into the [address book]({% link customers/account-dashboard-address-book.md %}) from the customer’s account, or during [checkout]({% link sales/checkout-step1-shipping.md %}).
+The Customer Address Attribute determines the properties of street addresses that are entered into the [address book]({% link customers/account-dashboard-address-book.md %}) from the customer’s account, or during [checkout]({% link sales/checkout-process.md %}).
```
Comment (hickskenh) [inline src/stores/attributes-customer-address.md:6]:
```suggestion
The Customer Address Attribute determines the properties of street addresses that are entered into the [address book]({% link customers/account-dashboard-address-book.md %}) from the customer’s account or during [checkout]({% link sales/checkout-process.md %}).
```

**Review by hickskenh — APPROVED** [review]
Approved after your review.

File: src/magento/prelaunch-checklist.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ After you complete the design, development, and testing of your store, check the
 ## Sales &amp; Marketing Settings
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/magento/prelaunch-checklist.md:20]: Yes!

**Comment by leslietilling** [comment]
running tests

**Pull request merged**